src/Raven.Server/Config/Categories/DatabaseConfiguration.cs

```diff
@@ -34,5 +34,10 @@ public class DatabaseConfiguration : ConfigurationCategory
         [ConfigurationEntry("Raven/Databases/FrequencyToCheckForIdleDatabasesInSec")]
         [ConfigurationEntry("Raven/Databases/FrequencyToCheckForIdleDatabases")]
         public TimeSetting FrequencyToCheckForIdle { get; set; }
+
+        [Description("The maximum number of documents to keep track of, in the huge documents list")]
```

Comment:
Wrong comment and value.
What we need here is the max size after which we'll warn that the document is too large.
By default, it should be 5MB